@here/harp-omv-datasource/lib/OmvRestClient.ts

```diff
@@ -362,6 +383,9 @@ export class OmvRestClient implements DataProvider {
             case APIFormat.XYZJson:
                 path += ".json";
                 break;
+            case APIFormat.XYZSpace:
+               path += ".mvt";
```

Comment:
maybe you can set the whole path here (including the .mvt) and remove the if above

Reply:
I went with @Astrak 's approach